internal/util/update/fastforward_test.go

```diff
@@ -44,7 +44,7 @@ func TestUpdate_FastForward(t *testing.T) {
 				WithKptfile(
 					pkgbuilder.NewKptfile().
 						WithUpstream("github.com/GoogleContainerTools/kpt", "/", "master", "fast-forward").
-						WithUpstreamLock(),
+						WithUpstreamLock("github.com/GoogleContainerTools/kpt", "/", "master", "abc123"),
```

Comment:
Better to make `github.com/GoogleContainerTools/kpt` and `master` constants. It will be easier to switch to `main`

Reply:
I moved `github.com/GoogleContainerTools/kpt` into a constant. I don't think we need to use a constant for `master`, since there is nothing special about the branch names in the tests. We will not have to make any changes for `master` vs `main`.